packages/wallet/src/services/TransactionReemitter.ts

```diff
@@ -81,6 +86,7 @@ export const createTransactionReemitter = ({
   );
 
   return rollback$.pipe(
+    tap((tx) => logger.debug(`Transaction ${tx.id} was rolled back`)),
```

Comment:
I think it's better to log at source (where rollback$ is created in TransactionsTracker)

Reply:
Done in [bd1f8c0](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/bd1f8c0a0a499a2060572ee911828d2848edbb9d)